src/main/java/embl/ebi/variation/commons/models/converters/data/VariantReadConverter.java

```diff
@@ -0,0 +1,26 @@
+package embl.ebi.variation.commons.models.converters.data;
+
+import com.mongodb.DBObject;
+import org.opencb.biodata.models.variant.Variant;
+import org.opencb.opencga.storage.core.variant.VariantStorageManager;
+import org.opencb.opencga.storage.mongodb.variant.DBObjectToVariantConverter;
+import org.opencb.opencga.storage.mongodb.variant.DBObjectToVariantSourceEntryConverter;
+import org.opencb.opencga.storage.mongodb.variant.DBObjectToVariantStatsConverter;
+import org.springframework.core.convert.converter.Converter;
+
+/**
+ * Created by tom on 21/09/16.
+ */
+public class VariantReadConverter implements Converter<DBObject, Variant> {
```

Comment:
In naming this class I was following the convention used in the [documentation](http://docs.spring.io/spring-data/mongodb/docs/current/reference/html/#mongo.custom-converters.reader). Should I still change the name?